README.md

```diff
@@ -61,11 +61,11 @@ import * as PXBThemes from '@pxblue/react-native-themes';
 
 The default theme structure for React Native Paper components does not offer us enough control to make some components look exactly the way they should for PX Blue applications. Because of this, we have extended the default theme type definition (see below) and created wrapper components with the correct styles to use in place of some of the standard React Native Paper components.
 
-In order for these components to look correct in your application, you should use the [PX Blue wrapper components](https://github.com/pxblue/react-native-component-library/blob/master/components/src/themed/README.md) in place of the respective components from React Native Paper.
+In order for these components to look correct in your application, you should use the [PX Blue wrapper components](https://github.com/brightlayer-ui/react-native-component-library/blob/master/components/src/themed/README.md) in place of the respective components from React Native Paper.
```

Comment:
Can you update PX Blue wrapper components to Brightlayer UI wrapper components ?

Reply:
Done.